src/main/java/com/structurize/compat/optifine/OptifineCompat.java

```diff
@@ -0,0 +1,188 @@
+package com.structurize.compat.optifine;
```

Comment:
Can you please run our formatter on this one?

Reply:
Did I not do that?

Reply:
nope =D